### PR TITLE
elasticsearch 2.3

### PR DESCRIPTION
**Pull request by maxpsq**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/elasticsearch@2.3.rb [hunk 1]
```diff
@@ -0,0 +1,116 @@
+class ElasticsearchAT23 < Formula
+  desc "Distributed search & analytics engine"
+  homepage "https://www.elastic.co/products/elasticsearch"
+  url "https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.3.5/elasticsearch-2.3.5.tar.gz"
+  sha256 "1119a8c18620b98c4b85261318663a1f26dea92a26f34dfeb7f813fb7cbb468a"
+  revision 1
```
Comment (MikeMcQuaid) [inline Formula/elasticsearch@2.3.rb]: Can remove this for new formulae.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
Is this version still supported by upstream?

File: Formula/elasticsearch@2.3.rb (hunk 1 above)
Comment (maxpsq) [inline Formula/elasticsearch@2.3.rb]: I don't get what can be removed...

Comment (JCount) [inline Formula/elasticsearch@2.3.rb]: The revision line can be removed.

**Comment by maxpsq** [comment]
Yes, this version is still supported.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**